README.md

```diff
@@ -64,18 +64,17 @@ Thank you to all our backers! 🙏 [[Become a backer](https://opencollective.com
 
 ### Sponsors
 
-Support this project by becoming a sponsor. Your logo will show up here with a link to your website. [[Become a sponsor](https://opencollective.com/openemr#sponsor)]
-
-<a href="https://opencollective.com/openemr/sponsor/0/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/0/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/1/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/1/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/2/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/2/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/3/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/3/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/4/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/4/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/5/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/5/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/6/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/6/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/7/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/7/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/8/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/8/avatar.svg"></a>
-<a href="https://opencollective.com/openemr/sponsor/9/website" target="_blank"><img src="https://opencollective.com/openemr/sponsor/9/avatar.svg"></a>
+Support this project on OpenCollective. [[Become a sponsor](https://opencollective.com/openemr#sponsor)]
+
+<a href="https://opencollective.com/openemr#sponsors" target="_blank"><img src="https://opencollective.com/openemr/sponsors.svg?width=890"></a>
+
+<a href="https://github.com/sponsors/openemr" target="_blank">
+  <svg aria-hidden="true" height="16" viewBox="0 0 16 16" version="1.1" width="16" data-view-component="true" class="octicon octicon-heart icon-sponsor mr-1 v-align-middle color-fg-sponsors anim-pulse-in">
```

Comment:
is there supposed to be something here that shows up in the readme?

Reply:
sorry for the echo. github has been acting odd today

Reply:
it should show this little heart which shows up in my local
![Screenshot from 2022-06-08 12-04-31](https://user-images.githubusercontent.com/5618372/172664706-d25350a0-d144-476c-9e41-51f19626e7a2.png)

Reply:
might not appear on github, thanks for noticing

Reply:
maybe these latest changes are too harsh and it was nice to see some avatars at the bottom but adding the yml for funding hopefully organizes it better